src/menu/index.tsx

```diff
@@ -10,6 +10,8 @@ export interface MenuProps {
 }
 
 const Menu = ({ className, children, withSeparators = false }: MenuProps) => (
+  /* TODO: BBC-7416 fix a11y issue */
+  // eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role
   <nav className={cc(className)} role="menu">
```

Comment:
This one is weird, no?

Reply:
Following this spec:
https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html
the `role="menu"` should be applied on an `ul` element.

Reply:
We overuse the <nav> in general (I'm trying to remove them gradually from kairos).
In short, a <nav> is non focusable and only to describe content while the menu role is attached to an element that will receive focus and keys (to open/close/up/down in the menu) hence the valid warning.